MetaMask/metamask-extension
# UX: updated placeholder for erc-1155 image

Update placeholder image for ERC-1155 Nft when src of image is null. 

Note: When Src is null, we are going to display the Default image without show button

* Fixes #20849 

## Screenshots/Screencaps

### After

1. When importing ERC-1155 image and src is null
![Screenshot 2023-09-18 at 2 39 48 PM](https://github.com/MetaMask/metamask-extension/assets/39872794/504ad6b8-eeb3-4936-b636-c79c12641f23)

## Manual Testing Steps

1. Enable ipfs gateway toggle and display NFT media toggle in privacy setting page.
2. Go to testdapp, deploy an ERC1155 contract and batch mint tokens.
3. Go to MM and import the minted token.
4. If src is null, the default image should be shown without the show button.


## Pre-merge author checklist


## Pre-merge reviewer checklist


If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.
